### DISCUSSION TITLE
求助给原始文件上传按钮添加样式的方法

### DISCUSSION THREAD

**Discussion by Nyazira** [layui/layui#1854] [问答]
我不知道如何给<input type="file">标签添加样式，开发文档中的<input type="file">也没有样式，如果给<input type="file">添加class="layui-btn"会变得很奇怪
```
<input type="file" name="file" id="ID-upload-demo-form-files" class="layui-btn">
```
![image](https://github.com/layui/layui/assets/26781505/c3ab9dd7-cf5d-49f9-84f2-5c25fe44d7cc)
有什么办法能让<input type="file">变得和普通按钮一样的样式？

**Comment by Sight-wcg** [comment]
```
input[type="file"]::-webkit-file-upload-button{
   -webkit-appearance: none;
   border: 1px solid transparent;
   background-color: #16baaa;
   color: #fff;
}
```
